airbyte-integrations/connectors/destination-bigquery/src/test/kotlin/io/airbyte/integrations/destination/bigquery/BigQueryDatabaseHandlerTest.kt

```diff
@@ -26,20 +26,15 @@ class BigQueryDatabaseHandlerTest {
     fun `billing errors are wrapped as ConfigErrorException`() {
```

Comment:
If you're testing co-routines I believe it should be `...() runTest {`

Reply:
nvm you shouldn't need `runTest` or `coEvery` because non of this is suspend

Reply:
switched back to `every`